src/main/java/no/unit/nva/cristin/projects/PresentationConverter.java

```diff
@@ -1,13 +1,21 @@
 package no.unit.nva.cristin.projects;
 
+//import static nva.commons.core.attempt.Try.attempt;
+
 import java.util.ArrayList;
 import java.util.Optional;
 import java.util.TreeMap;
 import java.util.stream.Collectors;
+import no.unit.nva.cristin.projects.model.cristin.CristinProject;
+//import nva.commons.core.JsonUtils;
```

Comment:
I would suggest deleting the commented out code in this PR, because it will be hard for reviewers to check if later the code was deleted.